deep-learning/_posts/2019-01-22-FCN_MultiStepRL_ImageProcessing.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: review
+title: "Fully Convolutional Network with Multi-Step Reinforcement Learning for Image Processing"
+tags: deep-learning reinforcement-learning CNN denoising
+author: "Antoine Théberge"
+cite:
+    authors: "Ryosuke Furuta, Naoto Inoue, Toshihiko Yamasaki"
+    title:   "Fully Convolutional Network with Multi-Step Reinforcement Learning for Image Processing"
+    venue:   "Arxiv"
+pdf: "https://arxiv.org/abs/1811.04323"
+---
+
+## Introduction
+
+Reinforcement learning methods are typically used to control robots and have only recently been used for image processing purposes. The authors of the paper do the latter by assigning an agent per pixel, where the agent changes the pixel value by taking an action. They apply the method to three image processing tasks: Image denoising, image restoration and local color enhancement. They claim to achieve comparable or better performances than state-of-the-art methods.
+
+## Formally
+
+They extend A3C by assigning an agent to each pixel of the image, where each agent trains a Policy Network to determine the best action to take and a Value Network that tries to approximate the future rewards. Of course, it would be computationally impractical to actually train an agent per pixel, therefore the authors used a Fully-Convolutional Network to have the networks share the same parameters.
```

Comment:
to have the ~networks~ **pixels** share the same parameters.

amirite?

Reply:
Uhhhhhh, will re-word.